credentials/alts/alts_test.go

```diff
@@ -409,6 +411,16 @@ func establishAltsConnection(t *testing.T, handshakerAddress, serverAddress stri
 	if !success {
 		t.Fatalf("c.UnaryCall() timed out after %v", defaultTestShortTimeout)
 	}
+	if got, want := peer.AuthInfo.AuthType(), "alts"; got != want {
```

Comment:
I'm not sure I fully understand these checks. IIUC, alts here is a result of calling new creds [function](https://github.com/grpc/grpc-go/blob/987df1309236a950d3f618eb5287279d81a7396e/credentials/alts/alts.go#L143)
Shouldn't we check the result of it instead?

Reply:
That's the top-level ALTS credentials API, and there are other tests in this file that check the result of that API.

This PR is modifying a helper function that is used during the e2e test for the ALTS credentials API. Specifically, in addition to checking that the ALTS handshake succeeds, we are also checking that the `AuthInfo` object produced from the ALTS handshake has the expected "alts" type.

Reply:
Added a clarifying comment, per offline request.